### DISCUSSION TITLE
Disable Grouped Notifications

### DISCUSSION THREAD

**Discussion by MSOB7YY** [ReVanced/revanced-discussions#179] [Question]
not sure if this is possible but it's really a pain in the ass

[reddit thread](https://www.reddit.com/r/youtube/comments/r92ei4/is_there_a_way_to_turn_off_notification_grouping/)

**Comment by diabolo98** [comment]
Yes, it's more a nuisance than a feature.

**Comment by MSOB7YY** [comment]
there is no progress on this?
